frontend/src/data/tasks.json

```diff
@@ -1960,7 +1960,7 @@
-        "descriptionFr": "Il est normal de ne pas savoir quand vous voulez prendre votre retraite ! Choisir de se retirer du monde du travail et recevoir ses pensions publiques sont deux choses distinctes. Cependant, il est important de décider quand vous voulez recevoir vos pensions publiques. Avoir un plan avant de prendre sa retraite peut vous procurer la tranquillité d'esprit.\n\nVous n'avez pas besoin d'arrêter de travailler pour demander vos pensions. Le RPC et la SV vous permettent de choisir quand commencer votre pension de retraite, et chaque mois que vous reportez augmente votre montant mensuel. Le montant mensuel le plus élevé que vous pouvez recevoir est atteint à 70 ans, après quoi il n'y a aucun avantage à attendre. Si vous avez besoin d'argent plus tôt, vous pouvez commencer à recevoir votre pension de retraite du RPC dès l'âge de 60 ans, mais avec une réduction permanente. Pour toucher votre pension de la SV, vous devez être âgé d'au moins 65 ans.\n\nSi vous avez un faible revenu ou une mauvaise santé, vous devriez commencer lorsque vous êtes éligible.",
+        "descriptionFr": "Il est normal de ne pas savoir quand vous voulez prendre votre retraite ! Choisir de se retirer du monde du travail et recevoir ses pensions publiques sont deux choses distinctes. Cependant, il est important de décider quand vous voulez recevoir vos pensions publiques. Avoir un plan avant de prendre sa retraite peut vous procurer la tranquillité d'esprit.\n\nVous n'avez pas besoin d'arrêter de travailler pour demander vos pensions. Le RPC et la SV vous permettent de choisir quand commencer votre pension de retraite, et chaque mois que vous reportez augmente votre montant mensuel. Le montant mensuel le plus élevé que vous pouvez recevoir est atteint à 70 ans, après quoi il n'y a aucun avantage à attendre. Si vous avez besoin d'argent plus tôt, vous pouvez commencer à recevoir votre pension de retraite du RPC dès l'âge de 60 ans, mais avec une réduction permanente. Pour toucher votre pension de la SV, vous devez être âgé d'au moins 65 ans.\n\nSi vous avez un faible revenu ou une mauvaise santé, vous devriez commencer lorsque vous êtes admissible.",
```

Comment:
```suggestion
        "descriptionFr": "Il est normal de ne pas savoir quand vous voulez prendre votre retraite! Choisir de se retirer du monde du travail et recevoir ses pensions publiques sont deux choses distinctes. Cependant, il est important de décider quand vous voulez recevoir vos pensions publiques. Avoir un plan avant de prendre sa retraite peut vous procurer la tranquillité d'esprit.\n\nVous n'avez pas besoin d'arrêter de travailler pour demander vos pensions. Le RPC et la SV vous permettent de choisir quand commencer votre pension de retraite, et chaque mois que vous reportez augmente votre montant mensuel. Le montant mensuel le plus élevé que vous pouvez recevoir est atteint à 70 ans, après quoi il n'y a aucun avantage à attendre. Si vous avez besoin d'argent plus tôt, vous pouvez commencer à recevoir votre pension de retraite du RPC dès l'âge de 60 ans, mais avec une réduction permanente. Pour toucher votre pension de la SV, vous devez être âgé d'au moins 65 ans.\n\nSi vous avez un faible revenu ou une mauvaise santé, vous devriez commencer lorsque vous êtes admissible.",
```

(removed the space before the exclamation point -- see https://www.btb.termiumplus.gc.ca/tcdnstyl-chap?lang=fra&lettr=indx96&info0=17.07)